torch/distributions/multinomial.py

```diff
@@ -88,7 +89,7 @@ def param_shape(self):
 
     def sample(self, sample_shape=torch.Size()):
         sample_shape = torch.Size(sample_shape)
-        samples = self._categorical.sample(torch.Size((self.total_count,)) + sample_shape)
+        samples = self._categorical.sample(torch.Size((int(self.total_count),).__iter__()) + sample_shape)
```

Comment:
I was wondering if we can remove `__iter__()` seems like `(int(val),)` should return a `Tuple[int]` type. yes?

Reply:
You are right! I have removed it.

Reply:
That's correct! Will update